lib/selenium-cucumber/methods/click_elements_methods.rb

```diff
@@ -1,10 +1,23 @@
 require_relative 'required_files'
 
-
 def click(access_type, access_name)
   $driver.find_element(:"#{access_type}" => "#{access_name}").click
 end
 
+def click_by_text(access_type, access_name, text)
```

Comment:
how about checking the text present in page source?
$driver.page_source.include? text